### DISCUSSION TITLE
After machine reboot, there is no log file for recidive jail, so fail2ban does not work

### DISCUSSION THREAD

**Discussion by simon19900** [fail2ban/fail2ban#3857] [Q&A] [answered]
sorry for my poor english,
My Fail2Ban v0.11.2
I set up the [recivide] jail and this works fine with fail2ban already running. 
However, whenever my machine restarts, this error occurs because the fail2ban.log saved under /var/log/ does not exist, fail2ban does not run.
Because fail2ban can't run, it can't automatically generate fail2ban.log, so the error loop keeps coming.
I had to turn off the recidive jail and wait for fail2ban to run and generate fail2ban.log before activating it, because I needed the recidive jail and there were always people who were attacking me relentlessly. 
Manually ban the IP address but cannot set the bantime.
##
root@ImmortalWrt:~ fail2ban-client -t
2024-10-15 16:17:26,031 fail2ban                [23901]: ERROR   Failed during configuration: Have not found any log file for recidive jail
2024-10-15 16:17:26,032 fail2ban                [23901]: ERROR   ERROR: test configuration failed
root@ImmortalWrt:~ fail2ban-client restart
2024-10-15 16:17:38,784 fail2ban                [24021]: ERROR   Failed to access socket path: /var/run/fail2ban/fail2ban.sock. Is fail2ban running?
2024-10-15 16:17:38,949 fail2ban                [24021]: ERROR   Failed during configuration: Have not found any log file for recidive jail
##
Excuse me:
1. Is it possible to solve the problem of not being able to run fail2ban because the fail2ban.log does not exist and cannot enable recidive jail?
2. Is it because of the var/directory?
3. Does the new version have the function of delaying the start of jail?

**Comment by sebres** [comment, marked as answer]
1. The issue with non-existing log is know and will be targeted in #1379
2. No, the `/var/log` directory is alright (it is default for log-files and normally persistent directory), I guess a logrotation may remove it.
3. No, the issue is still open, but... recidive jail is a rudiment, see https://github.com/fail2ban/fail2ban/issues/3613#issuecomment-1776879230

Basically there are few possibilities to circumvent this:
- either log fail2ban to journal (and switch recidive jail to `systemd` backend);
- or recreate the fail2ban.log in postrotate of logrotate or with cron;
- or use `bantime.increment` instead, as suggested above;
